packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -189,10 +189,26 @@ export const FilterList: React.FunctionComponent = () => {
 export const Table: React.FunctionComponent = () => {
   const [pageSize, setPageSize] = useState(10)
   const [pageIndex, setPageIndex] = useState(0)
+
   const [selectedItems, setSelectedItems] = useState<any[]>([])
   const [sortField, setSortField] = useState('firstName')
```

Comment:
Sort field should no longer be static. Should be empty by default, and can be any of the sortable fields once the user clicks on one of the sortable columns.